astropy/timeseries/periodograms/lombscargle/implementations/fast_impl.py

```diff
@@ -24,13 +24,13 @@ def lombscargle_fast(t, y, dy, f0, df, Nf,
     fit_mean : bool (default=True)
         If True, then compute the floating-mean periodogram; i.e. let the mean
         vary with the fit.
-    normalization : string (optional, default='standard')
+    normalization : str, optional
```

Comment:
do mention the default in the description below

Reply:
As discussed for the other one, I won't do this in this PR.